packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.tsx

```diff
@@ -365,32 +365,44 @@ export const ManageVersions = (props: ManageVersionsProps) => {
         log={log}
       >
         <div>
-          <HorizontalTabs
-            labels={[stringsOverrides.namedVersions, stringsOverrides.changes]}
-            activeIndex={_currentTab}
-            onTabSelected={(index) => changeTab(index)}
+          <Tabs.Wrapper
```

Comment:
This looks weird now

<img width="1448" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/aec33551-1c98-47a9-9ee1-abf7e1b5f448">

<img width="1463" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/8dfbe75a-6b39-4e87-a96e-b5bb82f1aac2">


We need to think either apply some spacing, set height to take full space too..
not sure here..

@mayank99 @FlyersPh9 any suggestions? We need to have tabs and table as content.. It will be used as main content on some web page..

Reply:
Before:
![image](https://github.com/iTwin/admin-components-react/assets/81580355/449d6cb3-0466-444a-a2fd-7aec089f856f)
![image](https://github.com/iTwin/admin-components-react/assets/81580355/67ec16dd-8427-4869-9a7c-ea1be157e7cb)

After:
<img width="1700" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/dc3f6d34-37a7-45d3-bec1-f2f15eb7dd71">
<img width="1592" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/da2ade22-3852-429f-a0b4-5e3e30d5e373">

Reply:
If you're using borderless tabs, they have always been transparent. There is no recent change in the tablist background afaict. You might have some custom CSS there.

I would recommend setting the tablist background to `var(--iui-color-background)`.